doc/040_backup.rst

```diff
@@ -50,6 +50,23 @@ still get a nice live status display. Be aware that the live status shows the
 processed files and not the transferred data. Transferred volume might be lower
 (due to de-duplication) or higher.
 
+On windows the ``--use-windows-vss`` option will use windows volume shadow copy
+service (VSS) when creating backups. Restic will transparently create a VSS
+snapshot for each volume that contains files to backup. Files are read from the
+VSS snapshot instead of the regular file. This allows to backup files that are
+exclusively locked by another process during the backup.
```

Comment:
Please adjust to:

~~~
On Windows, the ``--use-windows-vss`` option will use Windows' Volume Shadow Copy
Service (VSS) when creating backups. Restic will transparently create a VSS
snapshot for each volume that contains files to backup. Files are read from the
VSS snapshot instead of the regular filesystem. This allows to backup files that are
exclusively locked by another process during the backup.
~~~

Reply:
Fixed in https://github.com/restic/restic/pull/2274/commits/3a3a297fe427caa8a2b918b62b4c8a70185d0483